APP-MANAGER

```diff
@@ -10,6 +14,24 @@ CLI=$(basename "$0")
 arch="$HOSTTYPE"
 currentuser="$USER"
 
+# Determine sytem package manager
+function _system_package_manager_check() {
+	declare -A osInfo;
+	osInfo[/etc/redhat-release]=dnf
+	osInfo[/etc/arch-release]=pacman
+	osInfo[/etc/gentoo-release]=emerge
+	osInfo[/etc/SuSE-release]=zypper
+	osInfo[/etc/debian_version]=apt-get
+	osInfo[/etc/alpine-release]=apk
+	for f in ${!osInfo[@]};	do
+		if [[ -f $f ]];then
+			osPKGman="1"
```

Comment:
```
function _system_package_manager_check() {
    if command -v dnf &> /dev/null; then
        osPKGman="dnf"
    elif command -v pacman &> /dev/null; then
        osPKGman="pacman"
    elif command -v emerge &> /dev/null; then
        osPKGman="emerge"
    elif command -v zypper &> /dev/null; then
        osPKGman="zypper"
    elif command -v apt-get &> /dev/null; then
        osPKGman="apt-get"
    elif command -v apk &> /dev/null; then
        osPKGman="apk"
    else
        osPKGman="unknown"
    fi
}

```

do not over complicate

Reply:
oh, sorry... I was just working and testing this in my way, I've not seen the messages before https://github.com/ivan-hc/AM/commit/b944fd1a1cd1ff1d8ae59051a6bc0e2cc3d40220